test/OpenTelemetry.Exporter.Geneva.Tests/OpenTelemetry.Exporter.Geneva.Tests.csproj

```diff
@@ -33,7 +33,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="OpenTelemetry.Exporter.InMemory" Version="1.2.0-rc2" />
```

Comment:
Not blocking this PR, could these be extracted to one place?

Reply:
Done